### PR TITLE
Stop endianess conversion in parser

### PR DESCRIPTION
**Pull request by komaljai**

### CODE REVIEW

File: backends/ebpf/runtime/ebpf_kernel.h [hunk 1]
```diff
@@ -39,6 +39,9 @@ limitations under the License.
 
 #define load_byte(data, b) (*(((u8*)(data)) + (b)))
 #define load_half(data, b) bpf_ntohs(*(u16 *)((u8*)(data) + (b)))
+#define load_half_ne(data, b) (*(u16 *)((u8*)(data) + (b)))
+#define load_word_ne(data, b) (*(u32 *)((u8*)(data) + (b)))
+#define load_dword_ne(data, b) (*(u64 *)((u8*)(data) + (b)))
```
Comment (psivanup) [inline backends/ebpf/runtime/ebpf_kernel.h:48]: could move this set to below with a comment (e.g load without endian conversion)

**Review by psivanup — COMMENTED** [review]
Looks fine, Please add some comments.

File: backends/ebpf/runtime/ebpf_kernel.h (hunk 1 above)
Comment (komaljai) [reply backends/ebpf/runtime/ebpf_kernel.h:48]: okay

**Review by psivanup — APPROVED** [review]
Looks fine

**Pull request closed without merging**